recurly/__init__.py

```diff
@@ -1321,11 +1325,6 @@ class PlanRampInterval(Resource):
         'unit_amount_in_cents',
         'starting_billing_cycle'
     }
-    
-    def __str__(self):
```

Comment:
Just to understand, these methods are basically the "to-string" methods for these resource instances?

Reply:
yeah, because if you print for example an array of intervals you receive something like 

`Ramp Intervals: [<recurly.RampInterval object at 0x7f6543b90d60>, <recurly.RampInterval object at 0x7f6543b90910>]`